src/test/java/org/zalando/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -92,95 +62,18 @@
-    private final SchemaEvolutionService schemaEvolutionService = new ValidatorConfig()
-            .schemaEvolutionService();
-    private final AuthorizationValidator authorizationValidator = mock(AuthorizationValidator.class);
-
-    private MockMvc mockMvc;
 
-    public EventTypeControllerTest() throws IOException {
-    }
-
-    @Before
-    public void init() throws Exception {
-
-        final NakadiSettings nakadiSettings = new NakadiSettings(0, 0, 0, TOPIC_RETENTION_TIME_MS, 0, 60,
-                NAKADI_POLL_TIMEOUT, NAKADI_SEND_TIMEOUT, 0, NAKADI_EVENT_MAX_BYTES,
-                NAKADI_SUBSCRIPTION_MAX_PARTITIONS);
-        final PartitionsCalculator partitionsCalculator = new KafkaConfig().createPartitionsCalculator(
-                "t2.large", objectMapper, nakadiSettings);
-        when(timelineService.getDefaultTopicRepository()).thenReturn(topicRepository);
-        when(timelineService.getTopicRepository((Timeline) any())).thenReturn(topicRepository);
-        when(timelineService.getTopicRepository((EventTypeBase) any())).thenReturn(topicRepository);
-        when(transactionTemplate.execute(any())).thenAnswer(invocation -> {
-            final TransactionCallback callback = (TransactionCallback) invocation.getArguments()[0];
-            return callback.doInTransaction(null);
-        });
-
-        final EventTypeService eventTypeService = new EventTypeService(eventTypeRepository, timelineService,
-                partitionResolver, enrichment, subscriptionRepository, schemaEvolutionService, partitionsCalculator,
-                featureToggleService, authorizationValidator, timelineSync, transactionTemplate, nakadiSettings);
-
-        final EventTypeOptionsValidator eventTypeOptionsValidator =
-                new EventTypeOptionsValidator(TOPIC_RETENTION_MIN_MS, TOPIC_RETENTION_MAX_MS);
-        final EventTypeController controller = new EventTypeController(eventTypeService,
-                featureToggleService, eventTypeOptionsValidator, applicationService, nakadiSettings, settings);
-        doReturn(randomUUID).when(uuid).randomUUID();
-
-        final MappingJackson2HttpMessageConverter jackson2HttpMessageConverter =
-                new MappingJackson2HttpMessageConverter(objectMapper);
-
-        doReturn(true).when(applicationService).exists(any());
-        doReturn(SecuritySettings.AuthMode.OFF).when(settings).getAuthMode();
-        doReturn("nakadi").when(settings).getAdminClientId();
-        doReturn(true).when(featureToggleService).isFeatureEnabled(CHECK_PARTITIONS_KEYS);
-
-        mockMvc = standaloneSetup(controller)
-                .setMessageConverters(new StringHttpMessageConverter(), jackson2HttpMessageConverter)
-                .setCustomArgumentResolvers(new ClientResolver(settings, featureToggleService))
-                .setControllerAdvice(new ExceptionHandling())
-                .build();
+public class EventTypeControllerTest extends EventTypeControllerTestCase {
```

Comment:
Because EventTypeControllerTest is huge and I want to have another test which uses the same fields and methods, EventTypeAuthorizationTest